app/src/test/java/org/simple/clinic/util/scheduler/TrampolineSchedulersProvider.kt

```diff
@@ -2,8 +2,18 @@ package org.simple.clinic.util.scheduler
 
 import io.reactivex.Scheduler
 import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
 
-class TrampolineSchedulersProvider : SchedulersProvider {
+class TrampolineSchedulersProvider
+
+@Deprecated(
```

Comment:
Finally, I got confused a couple of times between TestSchedulersProvider and TrampolineSchedulersProvider usage 🚀